test/js/testRenderHerman.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
+const assert = require('assert');
+const del = require('del');
+const fs = require('fs');
+const Promise = require('bluebird');
+
+const prepareContext = require('../../lib/prepareContext');
+const renderHerman = require('../../lib/renderHerman');
+
+const access = Promise.promisify(fs.access);
+
+describe('renderHerman', function() {
+  before(function() {
+    this.dest = `${__dirname}/dest`;
+  });
+
+  afterEach(function() {
+    del.sync(`${this.dest}/*`);
+  });
+
+  it('does the herman things', function(done) {
```

Comment:
These `it` clauses were placeholders, as I didn't entirely know what I was asserting about the underlying code yet. Is there something better we can put in here?

Reply:
I'll fix this one; I don't see any others?

Reply:
I'll flag them.